arrow/flight/flightsql/column_metadata.go

```diff
@@ -63,23 +63,23 @@ type ColumnMetadata struct {
 }
 
 func (c *ColumnMetadata) findStrVal(key string) (string, bool) {
-	idx := c.Data.FindKey(CatalogNameKey)
+	idx := c.Data.FindKey(key)
```

Comment:
This is unrelated fix, right?

Reply:
yea, it was a metadata issue I came across that turned out to be small enough that it made sense to just incorporate